src/commands/project/create-variable-group.test.ts

```diff
@@ -375,6 +378,76 @@ describe("updateLifeCyclePipeline", () => {
     }
     expect(noFileError).toBeDefined();
   });
+
+  test("Should pass adding variable groups when bedrock file exists with empty variableGroups", async () => {
```

Comment:
@sarath-p do these two tests means "we should NOT add variable groups when a bedrock file already exists with an empty variable group?" and "we should NOT add variable groups when a bedrock file already exists with one variable group?" as far as i know, we should support adding multiple variable groups and add the list of them to each pipeline.

Reply:
No, it always replaces variable groups in the lifecycle pipeline from the list of variable groups in bedrock.yaml. The tests are simply checking whether code works when empty array and array with one variable group present in bedrock.yaml.